lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -244,7 +244,7 @@ def _transformed_cube(self, vals):
                 (minx, maxy, maxz)]
         return proj3d._proj_points(xyzs, self.M)
 
-    def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
+    def set_aspect(self, aspect, adjustable=None, anchor=None):
```

Comment:
Why did you delete `share`?

Reply:
Everything public must be deprecated before removal, if it indeed should be removed.

Reply:
I think deprecating `share` is the right thing to do, as the previous behavior wasn't doing anything AFAICT.